apps/event-worker/src/resource-scan/google/vpc.ts

```
@@ -16,61 +17,119 @@ import { getGoogleClient } from "./client.js";
-            {},
-          ),
-        }),
-      };
-    });
+            "google/project": project,
+            "google/self-link": network.selfLink,
+            "google/creation-timestamp": network.creationTimestamp,
+            "google/description": network.description,
+            ...network.peerings?.reduce(
+              (acc, peering) => ({
+                ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)